services/core-web/src/tests/components/modalContent/__snapshots__/AddReportModal.spec.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`AddReportModal renders properly 1`] = `
   <SlidingForms
     formContent={
       Array [
-        <Memo(Connect(ReduxForm))
+        <UNDEFINED
```

Comment:
UNDEFINED?

Reply:
ugh. hmmm... why did that happen in the snapshot...

Reply:
I've npm installed, and updated snapshots, nothing changed though. Thoughts?